devtools/create_project/create_project.cpp

```diff
@@ -1070,6 +1070,7 @@ const Feature s_features[] = {
 	{      "translation",               "USE_TRANSLATION", false, true,  "Translation support" },
 	{           "vkeybd",                 "ENABLE_VKEYBD", false, false, "Virtual keyboard support"},
 	{    "eventrecorder",          "ENABLE_EVENTRECORDER", false, false, "Event recorder support"},
+	{            "tracy",                  "ENABLE_TRACY", false, false, "Tracy profiler support"},
```

Comment:
This is also not yet done - will be finished if we can agree to include this - but `TRACY_ENABLE` must be given as define. That is not yet done here.